moment-range/index.d.ts

```diff
@@ -14,11 +14,8 @@ declare namespace momentRange {
         start: moment.Moment;
         end: moment.Moment;
 
-        constructor(range: string);
-        constructor(range: Date[]);
-        constructor(range: moment.Moment[]);
-        constructor(start: Date, end: Date);
-        constructor(start: moment.Moment, end: moment.Moment);
+        constructor(range: string | Date[] | moment.Moment[]);
+        constructor(start: Date | moment.Moment, end: Date | moment.Moment);
```

Comment:
I think you need the last 2 constructors. This would allow you to call the constructor with `Date` for `start` and `moment.Moment` for `end`. Which is different from before.